multipart/multipart.py

```diff
@@ -136,13 +136,14 @@ class MultipartState(IntEnum):
 LOWER_Z = b"z"[0]
 NULL = b"\x00"[0]
 
-
-# Lower-casing a character is different, because of the difference between
-# str on Py2, and bytes on Py3.  Same with getting the ordinal value of a byte,
-# and joining a list of bytes together.
-# These functions abstract that.
-def lower_char(c: int) -> int:
-    return c | 0x20
+# Mask for ASCII characters that can be http tokens.
+# Per RFC7230 - 3.2.6, this is all alpha-numeric characters 
```

Comment:
Can you add the link on the comment as well, please?

Reply:
Sure. I added a commit with the link.